DataFormats/Provenance/BuildFile.xml

```diff
@@ -1,5 +1,6 @@
 <use name="FWCore/Utilities"/>
 <use name="FWCore/Reflection"/>
+<use name="FWCore/ParameterSet"/>
```

Comment:
The `#include`'s of `FWCore/ParameterSet` look unnecessary (code compiles without), so I'll make a PR to remove them to avoid this dependence.

Reply:
Also, this would appear to add a circular dependence.